src/btouch.cs

```diff
@@ -441,8 +442,11 @@ static int Main2 (string [] args)
 			foreach (var def in defines)
 				cargs.Append ("-define:").Append (def).Append (' ');
 			cargs.Append (paths).Append (' ');
-			if (nostdlib)
+			if (nostdlib) {
 				cargs.Append ("-nostdlib ");
+				if (isCSC)
+					cargs.Append ("-noconfig ");
```

Comment:
would it break `mcs` ?

Reply:
This is kind of hack you should move it to .targets. The reason it works with mcs is that it does not have as bloated list of default assemblies as csc